pytripgui/controller/main_cont.py

```diff
@@ -196,10 +196,12 @@ def open_voxelplan(self, ctx_path):
 
         # Check if there is a VDX file with the same basename
         logger.debug("Check for VDX")
-        from pytrip.util import TRiP98FilePath
-        _b = TRiP98FilePath(ctx_path, ctx).basename
-        _n = TRiP98FilePath(ctx_path, ctx).name
-        vdx_path = ctx_path.replace(_n, _b) + '.vdx'
+        # from pytrip.util import TRiP98FilePath
+        # _b = TRiP98FilePath(ctx_path, ctx).basename
+        # _n = TRiP98FilePath(ctx_path, ctx).name
+        logger.debug("Check if '{:s}' exists...".format(ctx_path))
+        # TODO: Determine the way files will be found in directories
+        vdx_path = ctx_path.replace('.hed', '.vdx')
```

Comment:
`replace` will replace string anywhere in the path, also in parent directories.

Better would be to use something that doesn't require replacing:
```
vdx_path = TRiP98FilePath(ctx_path, ctx).dir_basename + '.hed'
```